meta-leda-distro/recipes-containers/nerdctl/nerdctl_git.bbappend

```diff
@@ -0,0 +1,21 @@
+# /********************************************************************************
+# * Copyright (c) 2022 Contributors to the Eclipse Foundation
+# *
+# * See the NOTICE file(s) distributed with this work for additional
+# * information regarding copyright ownership.
+# *
+# * This program and the accompanying materials are made available under the
+# * terms of the Apache License 2.0 which is available at
+# * https://www.apache.org/licenses/LICENSE-2.0
+# *
+# * SPDX-License-Identifier: Apache-2.0
+# ********************************************************************************/
+# Branch has been renamed from master to main in upstream repository
+
+inherit go
+
+SRC_URI:remove = "git://github.com/containerd/nerdctl.git;name=nerdcli;branch=master;protocol=https"
+
+# Appending will not work, as the main SRC_URI checkout will clear the target directory and then
+# the vendor.fetch folder will be missing. Hence, prepending the new url to the list of SRC_URIs.
+SRC_URI =+ "git://github.com/containerd/nerdctl.git;name=nerdcli;branch=main;protocol=https"
```

Comment:
Why =+ equation for SRC_URI and not to remove it before, just to override it

Reply:
SRC_URI is an array with lots of entries, see src_uri.inc of the nerdctl recipe. All the dependencies are listed in the SRC_URI array, but we need to fix only a single entry (the "main" SRC_URI entry for the nerdctl source code).

There is no syntax in BitBake for "replace" or "overwrite" a single entry in an array variable, but there is ":remove" and ":prepend" and ":append" syntax (or the respective += for append and =+ for prepend).